docs/data/system/experimental-api/extend-sx-prop/ExtendTheSxProp.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { createTheme, ThemeProvider, Box } from '@mui/system';
+
+const theme = createTheme({
+  unstable_sxConfig: {
+    // You can now use the background key in sx
+    // by providing direct values from the palette
+    background: {
+      themeKey: 'palette',
+    },
+  },
+  palette: {
```

Comment:
Maybe we should use something else than `palette`. I could easily envision people getting confused because the palette is there by default with Material UI and Joy UI. For our friends in the UK 😅

```suggestion
    background: {
      themeKey: 'colour',
    },
  },
  colour: {
```